metricbeat/module/kubernetes/state_container/_meta/data.json

```diff
@@ -0,0 +1,45 @@
+{
+  "@timestamp": "2017-05-04T17:46:11.030Z",
+  "beat": {
+    "hostname": "X1",
+    "name": "X1",
+    "version": "6.0.0-alpha1"
+  },
+  "kubernetes": {
+    "container": {
+      "id": "docker://91fdd43f6b1b4c3dd133cfca53e0b1210bc557c2ae56006026b5ccdb5f52826f",
+      "image": "gcr.io/google-containers/kube-addon-manager:v6.3",
+      "name": "kube-addon-manager",
+      "resource": {
+        "requests": {
+          "cpu": {
+            "nanocores": 5000000
+          },
+          "memory": {
+            "bytes": 52428800
+          }
+        }
+      },
+      "status": {
```

Comment:
I'm using `kube-state-metrics` nomenclature here, check https://github.com/elastic/beats/pull/4253/files/0f5f98da5eb5acf11055a8d8cf551f7b4eb1ed0e#diff-c70b508c24452e40e23c98c944027e25R54 to see the metric family names


This is how I would explain it:

State (from kube-state-metrics) means that we get access to the global state of the cluster (think of it as an snapshot in time). While this `status` metrics are one of the specific metrics of a container.